docs/src/main/asciidoc/building-native-image.adoc

```diff
@@ -786,10 +786,11 @@ $ ./mvnw clean package -Dquarkus.package.type=native-sources -Dquarkus.native.co
 cd target/native-sources
 docker run \
   -it \
+  --user $(id -ur):$(id -gr) \
   --rm \
-  --v $(pwd):/work \# <1>
+  -v $(pwd):/work \# <1>
   -w /work \# <2>
-  --entrypoint bin/sh \
+  --entrypoint /bin/sh \
```

Comment:
Is it related to this change?

Reply:
Yes, it's from the documentation part talking about using `native-sources`. Testing this PR requires using that command.